modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -7,7 +7,29 @@ class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
         def submit
-          render json: { status: 'OK' }
+          bgs_claim = find_bgs_claim!(claim_id: params[:id])
+
+          render json: [] && return unless bgs_claim
+
+          claim_empty = bgs_claim&.dig(:bnft_claim_dto).nil?
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if claim_empty
```

Comment:
```suggestion
          if bgs_claim&.dig(:bnft_claim_dto).nil?
            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
          end
```

Looking at v2/claims_controller#show, I think we mock that. We're currently looking for a lighthouse claim or BGS claim and then handle accordingly. Returning an empty array doesn't provide the consumer with much knowledge on the actual error.